src/main/java/net/sf/jabref/sql/importer/DBImporter.java

```diff
@@ -104,8 +104,9 @@
                         rsEntryType.getStatement().close();
                     }
 
-                    List<String> colNames = this.readColumnNames(conn);
```

Comment:
Does something like 
`List<String> colNames = this.readColumnNames(conn).stream().filter(!columnsNotConsideredForEntries.contains);`
works?